docs/rules/vue-caution/element-selectors-with-scoped.md

```diff
@@ -15,3 +15,54 @@ The benchmark results clearly indicate that **class selectors consistently outpe
 
 > [!IMPORTANT]
 > These findings suggest that using class selectors can lead to more efficient performance, especially in scenarios with a large number of elements. Therefore, for better performance in styling large-scale web applications **class selectors should be preferred** over element selectors.
+
+## 😱 Examples of code for which this rule will throw a warning
+
+::: warning
+Using element selectors within scoped styles can lead to performance issues and conflicts. The following example demonstrates how element selectors are being used incorrectly within scoped styles:
+:::
+
+```vue
+<style scoped>
+button {
+  background-color: blue;
+}
+
+div {
+  color: red;
+}
+
+/* Imagine 10_000+ element selectors */
```

Comment:
Isn't it about having 10k div-s on the page?

Reply:
updated in https://github.com/rrd108/vue-mess-detector/pull/133/commits/1758796668e9b58a815d3eb2e9bfa175dd06a0ec